lib/landable/configuration.rb

```diff
@@ -152,10 +153,25 @@ def untracked_paths
 
     def dnt_enabled
       return true if @dnt_enabled.nil?
-
       @dnt_enabled
     end
 
+    def hutch_queue
+      @hutch_queue ||= 'events'
+    end
+
+    def hutch_enable_confirms
```

Comment:
enable_hutch turns on the hutch subsystem. hutch_enable_confirms turns on  publisher_confirmations.  This is an option which increases message reliability, at the cost of throughput.